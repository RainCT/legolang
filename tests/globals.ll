global int t
global int g = 512
global string s = "Hello"
global int[] e
global float flo = 0.25

t = 27

def foo():
    t = 28

print s + " " + g + " " + t + " " + flo
foo()
print s + " " + g + " " + t + " " + flo
