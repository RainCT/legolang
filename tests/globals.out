Hello 512 27 0.25
Hello 512 28 0.25
